gradle/buildscript.gradle

```diff
@@ -17,5 +17,5 @@ repositories {
 }
 
 dependencies {
-    classpath "gradle.plugin.nl.javadude.gradle.plugins:license-gradle-plugin:0.12.1"
```

Comment:
This upgrades the gradle plugin for checking that there is a license header for each java file. They changed the classpath of the plugin, so that's why this line changed.